src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -1790,13 +1813,6 @@ namespace HVACMultiSpeedHeatPump {
         int OutNode;                          // Outlet node number in MSHP loop
         int ZoneInNode;                       // Zone inlet node number in the controlled zone for MSHP
         Real64 RhoAir;                        // Air density at InNode
-        static bool MyOneTimeFlag(true);      // Initialization flag
-        static Array1D_bool MyEnvrnFlag;      // Used for initializations each begin environment flag
-        static Array1D_bool MySizeFlag;       // Used for sizing MSHP inputs one time
-        static Array1D_bool MyCheckFlag;      // Used to obtain the zone inlet node number in the controlled zone
-        static Array1D_bool MyFlowFracFlag;   // Used for calculatig flow fraction once
-        static Array1D_bool MyPlantScantFlag; // used for finding on heat recovery plant loop
-        static Array1D_bool MyStagedFlag;     // used for finding on staged thermostat
```

Comment:
Pulled a _bunch_ of static arrays into the main component struct.  Eliminated all statics in here.